lib/fluent/plugin/out_logdna.rb

```diff
@@ -67,6 +67,8 @@ def gather_line_data(tag, time, record)
       line[:app] = record['_app'] || record['app']
       line[:app] ||= @app if @app
       line.delete(:app) if line[:app].nil?
+      line[:env] = record['env']
+      line.delete(:env) if line[:env].nil?
```

Comment:
It would arguably be more concise if we did `line[:env] = record['env'] unless record['env'].nil?` but it doesn't match what's below so oh well.